spec/support/request/authentication_workflow.rb

```diff
@@ -26,8 +26,8 @@ def login_to_admin_section
 
   # TODO: Should probably just rename this to create_user
   def create_enterprise_user( attrs = {} )
-    new_user = create(:user, attrs)
-    new_user.spree_roles = [] # for some reason unbeknown to me, this new user gets admin permissions by default.
+    new_user = build(:user, attrs)
+    new_user.spree_roles = [Spree::Role.find_or_create_by!(name: 'user')]
```

Comment:
This should work in master, shouldn't it?

Reply:
There are some other issues around enterprise roles in the 3-0 branch. Maybe it's best to wait until they're fully resolved?

Reply:
I personally would still move it to master but I don't know the other role issues. Once this is merged into the 3-0 branch, we are unlikely to remove it, I guess. Unless we rebase on master regularly and copy it to master one day. :woman_shrugging: Not a big deal either way.